app/views/orders/confirm.html.erb

```diff
@@ -0,0 +1,60 @@
+<div class="row">
+  <h1 class="header-box"> 
+    Order #<%= @order.id %> Confirmation 
+  </h1>
+</div>
+
+
+<div class="row">
+  <div class="col-sm-12">
+    Name: <strong><%= @order.cc_name %></strong><br>
+    E-mail: <%= @order.email_address %><br>
+    Mailing Address: <%= @order.mailing_address %><br>
+    Order Placed: <%= @order.placed_at.strftime("%B %d, %Y") %><br>
+    Order Status: <%= @order.status.capitalize %><br>
+    <%=  %><br>
```

Comment:
Is this intentionally left blank? If styling was the end goal, it's best to just add a class to this div and use CSS.